#ifndef __network_account_h__
#define __network_account_h__

#include "net/mm_packet.h"

struct KO_dog_network;

extern void network_account_callback_function_registration(struct KO_dog_network* p);

//tcp/////////////////////////////////////////////////////////////////////////////////




#endif//__network_account_h__